pkg/providers/version/version.go

```diff
@@ -0,0 +1,90 @@
+package version
```

Comment:
Can u help add the apache2 copyright claim?

Reply:
ok